### PR TITLE
feat(cb2-6718): remove dependabot and add snyk

### PR DESCRIPTION
**Pull request by leifkemp-bjss**
## CB2-6718 - Add Snyk and remove Dependandabot from existing Node repos

_Replacing Dependabot with Snyk_
[CB2-6718](https://dvsa.atlassian.net/browse/CB2-6718)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -2,5 +2,3 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npm run security-checks
-
-npx prettier --ignore-unknown --write $(git diff --name-only --staged | tr '\n' ' ') | grep -o '^\S*' | xargs -I {} git add {}
```
Comment (mcadenas-bjss) [inline .husky/pre-commit:6]: why was this removed?

**Review by mcadenas-bjss — CHANGES_REQUESTED** [review]
you need to bring your branch up to date with develop. `git fetch origin && git merge origin develop`

File: .husky/pre-commit (hunk 1 above)
Comment (leifkemp-bjss) [reply .husky/pre-commit:6]: Need to revert these, tried bringing it up to date with develop but the merge messed up, 'cause my node hadn't started.

**Comment by leifkemp-bjss** [comment]
Closing PR to revert the problems caused by merging the wrong branch
